facia/app/controllers/FaciaController.scala

```diff
@@ -45,6 +46,40 @@ trait FaciaController extends Controller with Logging with ExecutionContexts wit
   def renderFrontJson(id: String) = renderFront(id)
   def renderContainerJson(id: String) = renderContainer(id, false)
 
+  def renderSomeFrontContainers(path: String, num: String, offset: String, size: String) = MemcachedAction { implicit request =>
+    getSomeCollections(Editionalise(path, Edition(request)), num.toInt, offset.toInt).map { collections =>
```

Comment:
what is num (or offset) isn't an int?  should we use a match statement and return a bad request 400 if not?